src/items.rs

```diff
@@ -806,8 +804,7 @@ fn rewrite_explicit_self(explicit_self: &ast::ExplicitSelf, args: &[ast::Arg]) -
 
             // this hacky solution caused by absence of `Mutability` in `SelfValue`.
             let mut_str = {
-                if let ast::Pat_::PatIdent(ast::BindingMode::BindByValue(mutability), _, _)
-                        = args[0].pat.node {
+                if let ast::Pat_::PatIdent(ast::BindingMode::BindByValue(mutability), _, _) = args[0].pat.node {
```

Comment:
Apparently not? I didn't look in to why this is being formatted badly. I'm guessing we gave up too easily somewhere. Again, its only a new bug because this wasn't getting caught before because it's inside a match expression.